test/test_integration_list.py

```diff
@@ -56,7 +54,7 @@ def test_list_help_returns_help_screen(self):
     @integration_test
     def test_list_inspect_user_returns_info_on_user(self):
         self.invoke_cli(self.cli_auth_params,
-               ['policy', 'replace', 'root', self.environment.path_provider.get_policy_path("conjur")])
+               ['policy', 'replace', '-b', 'root', '-f', self.environment.path_provider.get_policy_path("conjur")])
         output = self.invoke_cli(self.cli_auth_params, ['list', '--inspect', '--kind', 'user'])
```

Comment:
inspect suppose to send a lot of data like role,client_ip.. where we verify this?

Reply:
inspect sends a lot of data that it would be hard to mock because it would take up a whole page so I tried to filter what is returned by adding those constraints kind=user and by only validating that that a small part of the json is in the returned output. Otherwise, like I said it would be ridiculously long